src/index.js

```diff
@@ -86,6 +86,7 @@ const next_level_message = (level) => {
   if (level === undefined) {
     return "The queue is empty.";
   }
+  twitch.notLurkingAnymore(level.submitter); // If we pull up a level, we should reset the lurking status
```

Comment:
`twitch.notLurkingAnymore(level.submitter);` would need to be replaced with `twitch.notLurkingAnymore(level.username);`.

Reply:
Resolved as requested.